docs/source/more_types.rst

```diff
@@ -884,33 +884,33 @@ Keys that aren't required are shown with a ``?`` in error messages:
    reveal_type(options)
 
 Totality also affects structural compatibility. You can't use a partial
-TypedDict when a total one is expected. Also, a total TypedDict is not
+``TypedDict`` when a total one is expected. Also, a total ``TypedDict`` is not
 valid when a partial one is expected.
 
 Supported operations
 --------------------
 
-TypedDict objects support a subset of dictionary operations and methods.
+``TypedDict`` objects support a subset of dictionary operations and methods.
 You must use string literals as keys when calling most of the methods,
 as otherwise mypy won't be able to check that the key is valid. List
 of supported operations:
 
-* Anything included in ``typing.Mapping``:
+* Anything included in :py:class:`~typing.Mapping`:
 
   * ``d[key]``
   * ``key in d``
   * ``len(d)``
   * ``for key in d`` (iteration)
-  * ``d.get(key[, default])``
-  * ``d.keys()``
-  * ``d.values()``
-  * ``d.items()``
+  * :py:meth:`d.get(key[, default]) <dict.get>`
+  * :py:meth:`d.keys() <dict.keys>`
+  * :py:meth:`d.values() <dict.values>`
+  * :py:meth:`d.items() <dict.items>`
 
-* ``d.copy()``
-* ``d.setdefault(key, default)``
-* ``d1.update(d2)``
-* ``d.pop(key[, default])`` (partial TypedDicts only)
-* ``del d[key]`` (partial TypedDicts only)
+* :py:meth:`d.copy() <dict.copy>`
+* :py:meth:`d.setdefault(key, default) <dict.setdefault>`
+* :py:meth:`d1.update(d2) <dict.update>`
+* :py:meth:`d.pop(key[, default]) <dict.pop>` (partial ``TypedDict``\s only)
+* ``del d[key]`` (partial ``TypedDict``\s only)
```

Comment:
Please advise whether referencing `dict` methods is useful here.

Reply:
Yeah, I think it is useful.